src/logic/activities.js

```diff
@@ -299,6 +299,38 @@ export const roundsMissingScorecards = wcif =>
     .filter(round => groupActivitiesAssigned(wcif, round.id))
     .filter(round => parseActivityCode(round.id).eventId !== '333fm');
 
+export const competitorsRegisteredForAnEventWithoutGroups = wcif => {
+  let persons = [];
+  wcif.persons.forEach(person => {
+    if (!person.registration || person.registration?.status !== 'accepted')
+      return;
```

Comment:
Nitpick: when `return` is in its own line, it's better to always put brackets, so that adding a line before `return` (for example `console.log(person)`) doesn't push `return` out of if and therefore change the behaviour.

```suggestion
    if (!person.registration || person.registration?.status !== 'accepted') {
      return;
    }
```